client/src/components/common/TextField/textdisabled.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import TextField from '@material-ui/core/TextField';
+import PropTypes from 'prop-types';
+import style from './style';
+
+export default function InputDisabled({ label, value }) {
+  const classes = style();
+  return (
+    <div className={classes.root}>
+      <TextField
+        disabled
+        id="outlined-disabled"
+        label={label}
+        defaultValue={value}
+        className={classes.container}
+        variant="outlined"
+      />
+    </div>
```

Comment:
there is no need for this file, cause we can use on component for text filed and add disabled property as props for it when needed , just delete it & change what is needed append on that .